Sources/RInAppMessaging/Constants.swift

```diff
@@ -71,6 +71,12 @@ internal enum Constants {
             static func setBackOffUpperBoundSeconds(_ bound: Int32) {
                 Randomized.backOffUpperBoundSeconds = bound
             }
+
+            static func setDefaults() {
+                Default.initialRetryDelayMS = Int32(10000)
+                Randomized.initialRetryDelayMS = Int32(60000)
+                Randomized.backOffUpperBoundSeconds = Int32(60)
```

Comment:
Maybe these 3 values above can be declared as static immutable:
```
static let defaultiInitialRetryDelayMS = Int32(10000)
static let defaultInitialRetryDelayMS = Int32(60000)
static let defaultBackOffUpperBoundSeconds = Int32(60)
```

Reply:
Could be. It sounds more proper but it's just for tests. I'm not sure if I should add more lines of code in this case. I think I'll toss a coin 🙂

Reply:
`setDefaults()`is only used in the tests target?

Reply:
yes. And also Tests are the only place where they are modified. I should add a comment